src/lib/remark-runner.ts

```diff
@@ -1,6 +1,7 @@
 import extensions from 'markdown-extensions';
 import remark from 'remark';
-import engine from 'unified-engine';
+import { Settings } from 'unified';
+import engine, { CallbackContext } from 'unified-engine';
```

Comment:
You can use the `unifiedEngine` function without mixing exported module and internal exported members
```suggestion
import { CallbackContext, unifiedEngine } from 'unified-engine';
```

Reply:
Apparently they are not exporting it.
Maybe you could do something like:
```suggestion
import unifiedEngine from 'unified-engine';
```
And then use `unifiedEngine.CallbackContext` instead of `CallbackContext`.